### PR TITLE
ci: add manual trigger to build apisix-runtime debian package

### PR DESCRIPTION
**Pull request by Revolyssup**
https://github.com/api7/apisix-build-tools/pull/392

### CODE REVIEW

File: .github/workflows/package-apisix-runtime-deb-openresty-1.21.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        description: 'Tag name for the release'
+        required: true
+        default: ''
+
+jobs:
+  build:
+    timeout-minutes: 60
+    env:
+      BUILD_APISIX_RUNTIME_VERSION: 1.1.2
+      ARCH: amd64
```
Comment (nic-6443) [inline .github/workflows/package-apisix-runtime-deb-openresty-1.21.yml:16]: this env is useless? we get the arch by `matrix.platform.arch`

File: .github/workflows/package-apisix-runtime-deb-openresty-1.21.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+          echo "ARCH: ${{ env.ARCH }}"
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install -y make ruby ruby-dev rubygems build-essential
+
+      - name: Build apisix-runtime deb
+        run: |
+          if [ "${{ matrix.platform.arch }}" == "arm64" ]; then
+            make package type=deb app=apisix-runtime runtime_version=${BUILD_APISIX_RUNTIME_VERSION} image_base=debian image_tag=bullseye-slim arch=linux/arm64/v8
```
Comment (nic-6443) [inline .github/workflows/package-apisix-runtime-deb-openresty-1.21.yml:40]: this already include openssl3, right?

File: .github/workflows/package-apisix-runtime-deb-openresty-1.21.yml (hunk 1 above)
Comment (Revolyssup) [reply .github/workflows/package-apisix-runtime-deb-openresty-1.21.yml]: removed it. it was previously used for some reason but wasn't removed. Now removed it

File: .github/workflows/package-apisix-runtime-deb-openresty-1.21.yml (hunk 2 above)
Comment (Revolyssup) [reply .github/workflows/package-apisix-runtime-deb-openresty-1.21.yml:40]: yes. If you look above, I have checked out openresty/1.21 branch and that branch builds openresty1.21 with openssl 3.0

**Review by nic-6443 — APPROVED** [review]

**Pull request merged**